Monika After Story/game/script-topics.rpy

```diff
@@ -5090,6 +5090,8 @@ label monika_chloroform:
     m 1eksdld "The easiest way to kidnap somebody is to just get them drunk, or drug them."
     m 1lksdla "Not that kidnapping somebody like that is easy, anyway."
     m 3eua "On that note, here's a safety tip."
+    if persistent._mas_pm_introvert:
+        m 3rksdla "I know you're probably not one to enjoy doing that often, but just in case..."
```

Comment:
```suggestion
        m 3rksdla "I know you're probably not one to enjoy doing this often, but just in case..."
```

Just a small grammatical thing, since this line appears before the line which "that" refers to.